src/css/list.scss

```diff
@@ -79,3 +79,8 @@ section:first-of-type {
   margin: 0;
   padding: 2rem 0;
 }
+
+// Scroll padding for sticky header
+:root.view--list {
+  scroll-padding-top: 3rem;
```

Comment:
I like it, but it's not [working well on Safari](https://bugs.webkit.org/show_bug.cgi?id=179379). We can probably still proceed with this solution, since the user can still scroll to the thing in question.